filesystem/wad.c

```diff
@@ -337,6 +337,10 @@ static wfile_t *W_Open( const char *filename, int *error )
 		return NULL;
 	}
 
+	LittleLongSW(header.ident);
```

Comment:
Spaces in parentheses. :)
It's fine to not have them in type casts and between consequent parentheses, but they should be everywhere else:
For example:
```
(T *)(expr + (expression2 * n))
```
should be formatted as:
```
(T *)( expr + ( expression2 * n ))
```

Sorry, don't have clang-format for this yet, but we probably will migrate to something similar later.